src/eduid_userdb/reset_password/db.py

```diff
@@ -111,11 +112,11 @@ def get_state_by_eppn(
         return None
 
     @staticmethod
-    def init_state(state: Mapping) -> Optional[Union[ResetPasswordEmailState, ResetPasswordEmailAndPhoneState]]:
+    def init_state(state: Dict) -> Optional[Union[ResetPasswordEmailState, ResetPasswordEmailAndPhoneState]]:
```

Comment:
Instead of changing return type above you can just use `state = dict(state)` if things lower in the stack wants a dict.

Reply:
:+1: